2023-11-09T15:03:34.6040617Z ##[group]Run tox -e py -- --cov-report=xml
2023-11-09T15:03:34.6041300Z [36;1mtox -e py -- --cov-report=xml[0m
2023-11-09T15:03:34.6084213Z shell: /usr/bin/bash -e {0}
2023-11-09T15:03:34.6084713Z env:
2023-11-09T15:03:34.6085231Z   pythonLocation: /opt/hostedtoolcache/Python/3.12.0/x64
2023-11-09T15:03:34.6086198Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.12.0/x64/lib/pkgconfig
2023-11-09T15:03:34.6087137Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.0/x64
2023-11-09T15:03:34.6088030Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.0/x64
2023-11-09T15:03:34.6088875Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.0/x64
2023-11-09T15:03:34.6089717Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.0/x64/lib
2023-11-09T15:03:34.6090436Z ##[endgroup]
2023-11-09T15:03:35.1923811Z py: install_deps> python -I -m pip install pytest pytest-cov pytest-mock
2023-11-09T15:03:38.9561660Z .pkg: install_requires> python -I -m pip install 'setuptools>=46.4.0' versioningit
2023-11-09T15:03:42.5135115Z .pkg: _optional_hooks> python /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta
2023-11-09T15:03:42.7225135Z .pkg: get_requires_for_build_sdist> python /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta
2023-11-09T15:03:42.8453324Z .pkg: get_requires_for_build_wheel> python /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta
2023-11-09T15:03:42.9297935Z .pkg: install_requires_for_build_wheel> python -I -m pip install wheel
2023-11-09T15:03:43.9799889Z .pkg: freeze> python -m pip freeze --all
2023-11-09T15:03:44.4678843Z .pkg: packaging==23.2,pip==23.3.1,setuptools==68.2.2,versioningit==2.2.1,wheel==0.41.3
2023-11-09T15:03:44.4686591Z .pkg: prepare_metadata_for_build_wheel> python /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta
2023-11-09T15:03:44.5805282Z .pkg: build_sdist> python /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta
2023-11-09T15:03:44.6959709Z py: install_package_deps> python -I -m pip install 'PyYAML>=5.0' 'click-loglevel~=0.2' 'click>=7.0' 'in-place~=0.4' 'pydantic~=1.7' 'python-dateutil~=2.7' 'python-dotenv<2.0,>=0.11' 'requests~=2.20'
2023-11-09T15:03:48.2000896Z py: install_package> python -I -m pip install --force-reinstall --no-deps /home/runner/work/tinuous/tinuous/.tox/.tmp/package/1/tinuous-0.6.1.post42+g637db65.tar.gz
2023-11-09T15:03:53.3401388Z py: freeze> python -m pip freeze --all
2023-11-09T15:03:53.8738501Z py: certifi==2023.7.22,charset-normalizer==3.3.2,click==8.1.7,click-loglevel==0.5.0,coverage==7.3.2,idna==3.4,in-place==0.5.0,iniconfig==2.0.0,packaging==23.2,pip==23.3.1,pluggy==1.3.0,pydantic==1.10.13,pytest==7.4.3,pytest-cov==4.1.0,pytest-mock==3.12.0,python-dateutil==2.8.2,python-dotenv==1.0.0,PyYAML==6.0.1,requests==2.31.0,six==1.16.0,tinuous @ file:///home/runner/work/tinuous/tinuous/.tox/.tmp/package/1/tinuous-0.6.1.post42%2Bg637db65.tar.gz#sha256=079319edf6aa43c1ebb7834469792cf8b0d0231acca9c06c59fbc2bb1c0a3183,typing_extensions==4.8.0,urllib3==2.0.7
2023-11-09T15:03:53.8743683Z py: commands[0]> tinuous --help
2023-11-09T15:03:54.1806980Z Usage: tinuous [OPTIONS] COMMAND [ARGS]...
2023-11-09T15:03:54.1807538Z 
2023-11-09T15:03:54.1807996Z   Download build logs from GitHub Actions, Travis, Appveyor, and CircleCI
2023-11-09T15:03:54.1808701Z 
2023-11-09T15:03:54.1808831Z Options:
2023-11-09T15:03:54.1809626Z   -V, --version                   Show the version and exit.
2023-11-09T15:03:54.1810626Z   -c, --config FILE               Read configuration from the given file
2023-11-09T15:03:54.1812035Z                                   [default: tinuous.yaml]
2023-11-09T15:03:54.1812974Z   -E, --env FILE                  Load environment variables from given .env
2023-11-09T15:03:54.1813742Z                                   file
2023-11-09T15:03:54.1814755Z   -l, --log-level [NOTSET|DEBUG|INFO|WARNING|ERROR|CRITICAL]
2023-11-09T15:03:54.1815556Z                                   Set logging level  [default: INFO]
2023-11-09T15:03:54.1816415Z   --help                          Show this message and exit.
2023-11-09T15:03:54.1816920Z 
2023-11-09T15:03:54.1817050Z Commands:
2023-11-09T15:03:54.1817417Z   fetch         Download logs
2023-11-09T15:03:54.1818217Z   fetch-commit  Download logs for a specific commit
2023-11-09T15:03:54.1818919Z   sanitize      Sanitize secrets in logs
2023-11-09T15:03:54.2273305Z py: commands[1]> pytest --cov-report=xml test
2023-11-09T15:03:56.5515222Z ============================= test session starts ==============================
2023-11-09T15:03:56.5516544Z platform linux -- Python 3.12.0, pytest-7.4.3, pluggy-1.3.0
2023-11-09T15:03:56.5517305Z cachedir: .tox/py/.pytest_cache
2023-11-09T15:03:56.5517883Z rootdir: /home/runner/work/tinuous/tinuous
2023-11-09T15:03:56.5518464Z configfile: tox.ini
2023-11-09T15:03:56.5518974Z plugins: mock-3.12.0, cov-4.1.0
2023-11-09T15:03:56.5519500Z collected 56 items / 1 error
2023-11-09T15:03:56.5519831Z 
2023-11-09T15:03:56.5520089Z ==================================== ERRORS ====================================
2023-11-09T15:03:56.5520986Z _____________________ ERROR collecting test/test_config.py _____________________
2023-11-09T15:03:56.5522180Z .tox/py/lib/python3.12/site-packages/_pytest/runner.py:341: in from_call
2023-11-09T15:03:56.5523066Z     result: Optional[TResult] = func()
2023-11-09T15:03:56.5524001Z .tox/py/lib/python3.12/site-packages/_pytest/runner.py:372: in <lambda>
2023-11-09T15:03:56.5525119Z     call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
2023-11-09T15:03:56.5526342Z .tox/py/lib/python3.12/site-packages/_pytest/python.py:531: in collect
2023-11-09T15:03:56.5527190Z     self._inject_setup_module_fixture()
2023-11-09T15:03:56.5528315Z .tox/py/lib/python3.12/site-packages/_pytest/python.py:545: in _inject_setup_module_fixture
2023-11-09T15:03:56.5529397Z     self.obj, ("setUpModule", "setup_module")
2023-11-09T15:03:56.5534321Z .tox/py/lib/python3.12/site-packages/_pytest/python.py:310: in obj
2023-11-09T15:03:56.5535401Z     self._obj = obj = self._getobj()
2023-11-09T15:03:56.5537088Z .tox/py/lib/python3.12/site-packages/_pytest/python.py:528: in _getobj
2023-11-09T15:03:56.5538193Z     return self._importtestmodule()
2023-11-09T15:03:56.5539248Z .tox/py/lib/python3.12/site-packages/_pytest/python.py:617: in _importtestmodule
2023-11-09T15:03:56.5540470Z     mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
2023-11-09T15:03:56.5541726Z .tox/py/lib/python3.12/site-packages/_pytest/pathlib.py:567: in import_path
2023-11-09T15:03:56.5542678Z     importlib.import_module(module_name)
2023-11-09T15:03:56.5543769Z /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/importlib/__init__.py:90: in import_module
2023-11-09T15:03:56.5545010Z     return _bootstrap._gcd_import(name[level:], package, level)
2023-11-09T15:03:56.5545857Z <frozen importlib._bootstrap>:1381: in _gcd_import
2023-11-09T15:03:56.5546478Z     ???
2023-11-09T15:03:56.5546931Z <frozen importlib._bootstrap>:1354: in _find_and_load
2023-11-09T15:03:56.5547566Z     ???
2023-11-09T15:03:56.5548097Z <frozen importlib._bootstrap>:1325: in _find_and_load_unlocked
2023-11-09T15:03:56.5548798Z     ???
2023-11-09T15:03:56.5549320Z <frozen importlib._bootstrap>:929: in _load_unlocked
2023-11-09T15:03:56.5549941Z     ???
2023-11-09T15:03:56.5550786Z .tox/py/lib/python3.12/site-packages/_pytest/assertion/rewrite.py:186: in exec_module
2023-11-09T15:03:56.5551758Z     exec(co, module.__dict__)
2023-11-09T15:03:56.5552269Z test/test_config.py:9: in <module>
2023-11-09T15:03:56.5553345Z     from tinuous.config import GHPathsDict, GitHubConfig
2023-11-09T15:03:56.5554402Z .tox/py/lib/python3.12/site-packages/tinuous/config.py:12: in <module>
2023-11-09T15:03:56.5555269Z     from .appveyor import Appveyor
2023-11-09T15:03:56.5556201Z .tox/py/lib/python3.12/site-packages/tinuous/appveyor.py:10: in <module>
2023-11-09T15:03:56.5557314Z     from dateutil.parser import isoparse
2023-11-09T15:03:56.5558364Z .tox/py/lib/python3.12/site-packages/dateutil/parser/__init__.py:2: in <module>
2023-11-09T15:03:56.5559486Z     from ._parser import parse, parser, parserinfo, ParserError
2023-11-09T15:03:56.5560680Z .tox/py/lib/python3.12/site-packages/dateutil/parser/_parser.py:50: in <module>
2023-11-09T15:03:56.5561586Z     from .. import tz
2023-11-09T15:03:56.5562436Z .tox/py/lib/python3.12/site-packages/dateutil/tz/__init__.py:2: in <module>
2023-11-09T15:03:56.5563309Z     from .tz import *
2023-11-09T15:03:56.5564102Z .tox/py/lib/python3.12/site-packages/dateutil/tz/tz.py:37: in <module>
2023-11-09T15:03:56.5565061Z     EPOCH = datetime.datetime.utcfromtimestamp(0)
2023-11-09T15:03:56.5567672Z E   DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.fromtimestamp(timestamp, datetime.UTC).
2023-11-09T15:03:56.5570045Z =========================== short test summary info ============================
2023-11-09T15:03:56.5571227Z ERROR test/test_config.py - DeprecationWarning: datetime.datetime.utcfromtime...
2023-11-09T15:03:56.5572344Z !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
2023-11-09T15:03:56.5573212Z =============================== 1 error in 1.63s ===============================
2023-11-09T15:03:56.6370806Z py: exit 2 (2.41 seconds) /home/runner/work/tinuous/tinuous> pytest --cov-report=xml test pid=1923
2023-11-09T15:03:56.6373028Z .pkg: _exit> python /opt/hostedtoolcache/Python/3.12.0/x64/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta
2023-11-09T15:03:56.6805224Z   py: FAIL code 2 (21.73=setup[18.97]+cmd[0.35,2.41] seconds)
2023-11-09T15:03:56.6806089Z   evaluation failed :( (21.83 seconds)
2023-11-09T15:03:56.7316513Z ##[error]Process completed with exit code 2.
